### PR TITLE
SD-98474 | [Madeira Madeira] Custom Identifier Improvement

### PR DESCRIPTION
**Pull request by emreisik95**
Partners can add custom identifiers to their payload. Just new field added.

## Testing

We matched the custom identifiers and sent it to our side, we checked we are getting data correctly.

- [x] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [x] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

**Comment by joe-ayoub-segment** [comment]
Hi @emreisik95 thanks for the PR. I'll schedule for review.

File: packages/destination-actions/src/destinations/insider/insider-properties.ts [hunk 1]
```diff
@@ -156,6 +156,14 @@ export const segment_anonymous_id: InputField = {
   default: { '@path': '$.anonymousId' }
 }
 
+export const custom_identifiers: InputField = {
+  label: 'Custom Identifiers',
+  type: 'object',
+  description: 'You can select your custom identifiers for the event.',
```
Comment (joe-ayoub-segment) [inline packages/destination-actions/src/destinations/insider/insider-properties.ts:162]: hi @emreisik95. 

2 questions: 

1. Should we default the UI to display key:value pairs?

2. Should we make the description a little more useful? Are there specific types of Custom Identifiers the user should know about? How should the user know what to place in this field? Can you provide an example or link to a doc?

Comment (emreisik95) [reply packages/destination-actions/src/destinations/insider/insider-properties.ts:162]: Yes, it should be default key:value pairs, we are accepting it as key:value. Also, it should be undefined if partner won't use it.

For the second question, I'll align with our PM's and answer.

**Comment by joe-ayoub-segment** [comment]
Hi @emreisik95 - I reviewed the PR. Looks good. Just one minor item pointed out. Please let me know if it's something you want to change or not?

**Comment by emreisik95** [comment]
> Hi @emreisik95 - I reviewed the PR. Looks good. Just one minor item pointed out. Please let me know if it's something you want to change or not?

We can keep continue like this. Thanks for your review.

**Review by joe-ayoub-segment — APPROVED** [review]

**Comment by joe-ayoub-segment** [comment]
hi @emreisik95 PR deployed

**Comment by emreisik95** [comment]
> hi @emreisik95 PR deployed

Thanks a lot, we are checking.

**Pull request merged**